bin/go-test

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eu
+
+if [ $# -ne 2 ]; then
+    echo "Usage: bin/${0##*/} [package to test] [Test ID]" >&2
+    exit
+fi
+
+go test -v ./tests/$1 -ginkgo.v -ginkgo.reportFile=$(pwd)/reports/$2-$1.junit.xml
```

Comment:
> -v

While the tests are running, the output feels a bit verbose. Every test is repeating the same headers with information on the control plane namespace, cluster domain, CLI path etc. It might be better to just output this information once, at the start of the entire test run. Also, maybe provide a way to enable/disable `-v`.

E.g.,

```
Unspecified linkerd2 control plane namespace - use default value "l5d-conformance" // repeated in every test case
Unspecified cluster domain - using default value "cluster.local" // repeated in every test case
Unspecified path to linkerd2 binary - using default value "/home/isim/.linkerd2/bin/linkerd" // repeated in every test case
linkerd2 binary exists in "/home/isim/.linkerd2/bin/linkerd"- skipping installation
=== RUN   TestLifecycle
Running Suite: Lifecycle
========================
Random Seed: 1594265290  // ?
Will run 1 of 1 specs

 `linkerd install` 
  can install a new control plane
  /home/isim/workspace/linkerd/linkerd2-conformance/tests/lifecycle/specs.go:18
```

Reply:
What does the `linkerd2 binary exists in "/home/isim/.linkerd2/bin/linkerd"- skipping installation` header that appear before the install test mean? Is the install test being executed after all?

Reply:
Hmm, the reason why those headers are repeating is because each time a new test suite (`_test.go`) file is run, the config YAML is being re-read and parsed again, as each test suite runs as a separate `go test` process. But I'm sure there's a workaround to just show that output once.

Reply:
> What does the linkerd2 binary exists in `"/home/isim/.linkerd2/bin/linkerd"- skipping installation` header that appear before the install test mean?

Ah, so that header appears when the tests try to download a linkerd2 binary. Before downloading, it attempts to check if a binary exists in the default `/.linkerd2` directory. If it does, it skips the binary downloading part, which is what "skipping installation" means. I just realised that its misleading, and can easily be confused with "skipping control plane installation". I should change the message to "skipping linkerd2 binary installation".

> Is the install test being executed after all?

Yep, this header actually has nothing to do with the control plane installation, as I mentioned above. The install tests would run regardless. I will change the message :smile: